bugbug/repository.py

```diff
@@ -77,6 +82,9 @@ def _transform(commit):
     # Covert to a list, as a set is not JSON-serializable.
     obj['types'] = list(obj['types'])
 
+    components_touched = ['::'.join(COMPONENTS[fl]) for fl in patch_data.keys() if COMPONENTS.get(fl)]
```

Comment:
Here too, you don't need to create this list.

Either move this in the following line (doing `obj['components'] = list(set('::'.join(...`), or you can make this an iterator by replacing the `[` with `(` and the `]` with `)`.